src/main/pages/che-7/hands-on-guidance/proc_importing-a-vs-code-extension-in-the-che-theia-editor.adoc

```diff
@@ -0,0 +1,60 @@
+// Module included in the following assemblies:
+//
+// using-a-visual-studio-code-extension-in-che
+
+// This module can be included from assemblies using the following include statement:
+// include::<path>/proc_proc_importing-a-visual-studio-code-extension-in-the-che-theia-editor-adoc.adoc[leveloffset=+1]
+
+// The file name and the ID are based on the module title. For example:
+// * file name: proc_doing-procedure-a.adoc
+// * ID: [id='proc_doing-procedure-a_{context}']
+// * Title: = Doing procedure A
+//
+// The ID is used as an anchor for linking to the module. Avoid changing
+// it after the module has been published to ensure existing links are not
+// broken.
+//
+// The `context` attribute enables module reuse. Every module's ID includes
+// {context}, which ensures that the module has a unique ID even if it is
+// reused multiple times in a guide.
+//
+// Start the title with a verb, such as Creating or Create. See also
+// _Wording of headings_ in _The IBM Style Guide_.
+[id="proc_importing-a-visual-studio-code-extension-in-the-che-theia-editor-adoc_{context}"]
+= Importing a VS Code extension in the Che Theia Editor
+
+This section describes how to install a Visual Studio (VS) Code extension from link:https://marketplace.visualstudio.com/vscode[VS Code marketplace] in your workspace when using Che Theia as the Che editor component.
+
+This method of installing the VS Code extensions is aimed at extensions that do not have runtime dependencies. To install extensions that require particular dependencies, such as a tool, an environment variable, a runtime server, see section xref:publishing-a-vs-code-extension-into-the-che-plug-in-registry_using-a-visual-studio-code-extension-in-che[Publishing a VS Code extension into Che Plugin Registry.]
+
+
+[discrete]
+== Prerequisites
+
+* Configure the Che Theia memory limit with 512 MB RAM or more to run the Theia editor. In addition, reserve another 512 MB RAM for the VS Code extensions.
```

Comment:
```suggestion
* Configure the Che Theia memory limit with 512 MB RAM or more to run the Theia editor. In addition, reserve an additional 512 MB RAM for the VS Code extensions.
```

Reply:
@supriya-bharadwaj Hello! I fix this by rewriting into this shape:
"Configure the Che Theia memory limit with 512 MB RAM or more to run the Theia editor. In addition, reserve sufficient memory for the VS code extensions. The specific amount of memory required depends on the extensions used."